docs/walkthroughs/connecting-device-to-service.adoc

```diff
@@ -0,0 +1,34 @@
+[[mobile-core-use-public-ip-addr]]
+= Mobile Core installation with public IP
+
+To use your local OpenShift cluster for mobile development using a real device rather than an emulator, you will need to specify the public IP address of your local machine.
+
+NOTE: You can do this during the setup of mobile using the install script in this repo and provide following parameters
+```
+Use public IP yes/no?: _yes_
+Cluster IP local/public: <your-eth0-ip>|<your-wlan0-ip>
+```
+image::images/mobile-core-public-ip-setup.png[openshift-mobile-core-public-ip-setup]
+Find IP address of your local machine you can with this command ```ifconfig | grep inet```.
+Output should look like this
+```
+	inet 127.0.0.1 netmask 0xff000000
+	inet 10.40.2.105 netmask 0xfffffe00 broadcast 10.40.3.255
+	inet 10.200.156.50 netmask 0xfffff800 broadcast 10.200.159.255
+```
+You have two IP address available for WiFi and ethernet, in this example WiFi network and 10.200.156.50 ip address is used.
+When installation is finished and successful you should get same result as on image below
+
+image::images/mobile-core-public-ip-success.png[openshift-mobile-core-public-ip-success]
+Web console URL should contain your public IP address, in this case it should be `10.200.156.50`
+
+[[mobile-core-export-self-signed-cert]]
```

Comment:
Do we want to link to the existing doc for this? https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/self-signed-certs.adoc

Reply:
+1 we already have this doc and it negates the need for screenshots